pegasus/routes/poste_routes.rb

```diff
@@ -16,6 +16,11 @@
   end
   url = DB[:poste_urls].where(id: url_id).first
   pass unless url
+  if url[:deleted_at]
+    path = resolve_template('views', settings.template_extnames, 'page_not_available')
+    content = path ? document(path) : "Sorry, this page is no longer available.\n"
```

Comment:
When would `path` not exist?

Reply:
it's very unlikely (I think only if someone moved the haml file) but I'd rather have the check here than have the page give a 404 due to an exception